StringComparison/src/com/multithreading/EmployeeMain.java

```diff
@@ -7,9 +7,9 @@ public static void main(String[] args) {
 
 		Employee emp1 = new Employee();
 
-		EmployeeThread thread1 = new EmployeeThread("Child1", emp1, 201);
-		EmployeeThread thread2 = new EmployeeThread("Child2", emp1, 301);
-
+		EmployeeThread thread1 = new EmployeeThread("Child12", emp1, 201);
```

Comment:
why did you change the name of this class? please revert it back to child1. Please also add some comments in your code. do this do that....